multimedia/minidlna/patches/999-force-compatibility-android_channels.patch

```diff
@@ -0,0 +1,19 @@
+--- ./minissdp.c-orig	2020-11-24 19:53:50.000000000 +0100
++++ ./minissdp.c	2021-07-06 11:05:37.422070969 +0200
```

Comment:
Here I am missing details:
- Git commit author
- Git commit description

And if this patch was sent upstream or not. Because from your changes it is not clear why this is necessary at all.

Reply:
It does what I said, makes it compatible with the closed source `DLNA Channels`, it simply fails to get content from `minidlnad`. The warning was the only thing I could see in the logs, I tried commenting the conditional out and indeed it fixed the issue.

I haven't tried pushing it upstream, it is definitely worth the shot, that would be the really right place.

Reply:
You said it well in the pull request description. In the commit subject, there is just to be compatible with Android DLNA Channel, but the description in the patch itself is missing.